main.ts

```diff
@@ -2,30 +2,32 @@ import { Plugin } from "obsidian";
 
 export default class AutoHideCursorPlugin extends Plugin {
 	async onload() {
-		document.body.addEventListener(
+		this.setupEventListeners(document.body);
+
+		// this.registerEvent(
+		// 	this.app.workspace.on("active-leaf-change", (app) => {
+		// 		this.setupEventListeners(activeDocument.body);
+		// 	})
+		// );
```

Comment:
hey @joethei, i tried to use `activeDocument` but this approach doesn't work. any hints on how to deal with pop-out windows?